src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="form"
+      data-qa="big"
+    >
+      <label for="big-search" class="form__section-big">
+        <img class = "image big-image" 
+        src="./images/Search.svg" alt="big-image">
```

Comment:
```suggestion
        <img
          class="image big-image" 
          src="./images/Search.svg"
          alt="big-image"
        >
```

Reply:
The alt attribute should describe the image to which it is applied.
What does a "big-picture" mean?
What kind of picture?
`The magnifying glass icon` or something similar would be much better